spec.md

```diff
@@ -67,10 +67,6 @@ Several terms are used frequently in this document and warrant basic definitions
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
 
-## Use Cases
-
-TODO: more detail on workflows related
-
 ### Artifact Verification
```

Comment:
This may make the actual use cases like "Artifact Verification" appear under incorrect root header. Can we remove them as well, or otherwise just remove the TODO statement?